src/core/document/schema.pl

```diff
@@ -183,6 +180,23 @@
                   property: Property
               }.
 
+refute_schema_context_system_prefix(Schema, Prefix_Name, Witness) :-
+    (   xrdf(Schema, 'terminusdb://context', Prefix_Name, Prefix_Value)
+    ->  (   literal_to_string(Prefix_Value, String),
+            uri_has_protocol(String)
+        ->  fail
+        ;   Witness = witness{
+                          '@type': context_system_prefix_is_not_a_uri,
+                          prefix_name: Prefix_Name,
+                          prefix_value: Prefix_Value
```

Comment:
Prefix_Value could be a non-jsonifiable thing. You should make sure that values that go into witnesses can be converted into json.